src/util.c

```diff
@@ -231,7 +231,7 @@ void print_test_results(struct ntttcp_test_endpoint *tep)
 	}
 
 	if (tepr->final_cpu_ps->nproc == tepr->init_cpu_ps->nproc) {
-		ASPRINTF(&log, "cpu cores\t:%d", tepr->final_cpu_ps->nproc);
+		ASPRINTF(&log, "\t cpu cores\t:%d", tepr->final_cpu_ps->nproc);
```

Comment:
I think '\t' should not be here. The below items are all about CPU. The "cpu cores:" should be a parent item.

Reply:
done